src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/services/email/EmailSenderTest.java

```diff
@@ -37,12 +38,25 @@
     @Autowired
     private EmailSender emailSender;
 
-    @RegisterExtension
-    static GreenMailExtension greenMail = new GreenMailExtension(ServerSetupTest.SMTP);
```

Comment:
this means we don't use greenmail extension and should be removed from project test sources